examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<p>Mention synthetic data option here as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I included a markdown explaining what is our synthetic data generation engine and a code snippet to do so.